packages/camera-web/src/Camera/hooks/useUserMedia.ts

```diff
@@ -36,6 +36,15 @@ class InvalidStreamError extends Error {
  * @see useUserMedia
  */
 export enum UserMediaErrorType {
+  /**
+   * The camera stream couldn't be fetched because the web page does not have the permissions to access the camera.
+   * If both webpage and browser permissions are denied, this error will be returned.
+   */
+  WEBPAGE_NOT_ALLOWED = 'not_allowed_webpage',
+  /**
+   * The camera stream couldn't be fetched because the browser does not support the `getUserMedia` function.
+   */
+  BROWSER_NOT_ALLOWED = 'not_allowed_browser',
   /**
    * The camera stream couldn't be fetched because the web page does not have the permissions to access the camera.
```

Comment:
Change the description to : ``The camera stream couldn't be fetched, but the app is unable to know if it is because of the website or the browser not being allowed to have camera permission access. This error is usually returned on Firefox and other similar browsers where `navigator.permissions.query` is not supported.``